Persistence/Entities/PostMachine.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace shipping_service.Persistence.Entities
+{
+    public class PostMachine
+    {
+        public ulong Id { get; set; }
+        public string Name { get; set; }
```

Comment:
Why a post machine needs a name?

Reply:
To be able to show their names for the sender to choose. Full address might be long, so this name will be much simpler (e.g. full address is "Bakery street 5, London, England", but name could be just "Bakery mall").